contracts/Greeter.sol

```diff
@@ -0,0 +1,22 @@
+//SPDX-License-Identifier: Unlicense
```

Comment:
Can delete this

Reply:
I rate we leave it for now. it shows how to use the test setup